db/schema.rb

```diff
@@ -11,16 +11,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150316080645) do
```

Comment:
I think you have changed too many columns here ><

You can manually fix this or go back to the commits before rails 4.1 and setup the db, then migrate here again.